gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/test/java/io/gravitee/rest/api/service/impl/SearchEngineServiceTest.java

```diff
@@ -535,6 +537,25 @@ public void shouldFindWithoutEnvironmentId() {
         assertEquals(Arrays.asList("api-1", "api-2"), new ArrayList(matches.getDocuments()));
     }
 
+    @Test
+    public void shouldFindWithExcludedFilters() {
+        Map<String, Object> filters = new HashMap<>();
+        filters.put(FIELD_API_TYPE_VALUE, Arrays.asList("api-1", "api-2"));
+        Map<String, Collection<String>> excludedFilters = new HashMap<>();
+        excludedFilters.put(FIELD_DEFINITION_VERSION, Arrays.asList(DefinitionVersion.V1.getLabel(), DefinitionVersion.V4.getLabel()));
+        QueryBuilder<ApiEntity> apiEntityQueryBuilder = QueryBuilder
+            .create(ApiEntity.class)
+            .setFilters(filters)
+            .setExcludedFilters(excludedFilters);
+        SearchResult matches = searchEngineService.search(
+            new ExecutionContext(GraviteeContext.getCurrentOrganization(), null),
+            apiEntityQueryBuilder.build()
+        );
+        assertNotNull(matches);
+        assertEquals(1, 2, matches.getHits());
```

Comment:
are you sure of this syntax ??
I would suggest this instead:
```suggestion
        assertEquals(1, matches.getHits());
```